packages/core/src/docdb/explorer/dbElasticClusterNode.ts

```diff
@@ -21,7 +22,7 @@ export class DBElasticClusterNode extends DBResourceNode {
 
     constructor(
         public readonly parent: AWSTreeNodeBase,
-        readonly cluster: DBElasticCluster,
+        public cluster: DBElasticCluster,
```

Comment:
Did you intend to expose the raw cluster from this class?

Reply:
Yes, there may be occasions where I need to pull a value from the underlying data structure.